sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -349,7 +349,7 @@
     "\n",
     "When using AutoML for image tasks, you need to specify the model algorithms using the model_name parameter. You can either specify a single model or choose to sweep over multiple models. Please refer to the [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) for the list of supported model algorithms.\n",
     "\n",
-    "## 3.1. Configure the job through the image_classification() factory function\n",
+    "## Configure the job through the image_classification() factory function\n",
```

Comment:
I am a little confused on where the best place is to put this section. One other alternative is to move this under section 3.1 before the code cells. We can have "image_classification() function parameters:" and "set_limits() parameters:" in that section and have "set_sweep() parameters:" part in the 3.2 section.
@sharma-riti @gauravrajguru  what do you think ?